### ISSUE TITLE
Recommendations badges have two strings "Recommended"

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#8285]
In looking at #8284 it was noted that the markup contains two strings "Recommended":

<img width="563" alt="Recommended_add-ons_–_Add-ons_for_Firefox__en-US_" src="https://user-images.githubusercontent.com/1514/60964492-56ef4580-a30b-11e9-8c24-668ad1ee41e1.png">

One is visually hidden, but a screen-reader would presumably read out both. 

Since there's a visual string the visually-hidden one should be removed. This should be controllable by the alt prop in the Icon component: 

https://github.com/mozilla/addons-frontend/blob/a8e957ccda60ee61b4ec53f6fb776d25c345aa49/src/ui/components/Icon/index.js#L20-L22

**Comment by bobsilverberg** [comment]
@muffinresearch Do you think this is serious enough to merit a cherry-pick, or can we let it land next week?

**Comment by muffinresearch** [comment]
> @muffinresearch Do you think this is serious enough to merit a cherry-pick, or can we let it land next week?

It can wait until next week.

**Comment by AlexandraMoga** [comment]
Verified fixed on -dev. The visually-hidden string is no longer present:

![image](https://user-images.githubusercontent.com/31961530/61221048-4af6ef80-a720-11e9-930e-a56cdbc884ad.png)
